ScratchDjango/User/managers.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.hashers import make_password
+from django.contrib.auth.models import UserManager as DjangoUserManager
+from django.contrib.auth.password_validation import validate_password
+
+
+class UserManager(DjangoUserManager):
```

Comment:
please remove this file if it's not being used

Reply:
I remember now why it is created. For django default password validation, only defining it in the settings is not enough and we have to call the validate_password everytime to validate the password with the password validation settings defined in the settings